website/docs/getting-started/setup/instance-management/appsmithctl.mdx

```diff
@@ -215,14 +215,6 @@ After configuration, the restore command lists local and S3 bucket backups.
 
       b. Update the values:
 
-      * For the **Community Edition** with:
```

Comment:
@pratapaprasanna - Please verify this change. The feature is already available as part of the Business plan, so the instructions for the Community are not needed. Right?